frontend/src/js/components/tenants/expanded-tenant.tsx

```diff
@@ -11,18 +11,197 @@
+    color: theme.palette.primary.main,
+    fontSize: '11px',
+    fontWeight: 400
+  },
+  tenantTitle: {
+    fontSize: '17px',
+    fontWeight: 500
+  },
+  devLimitInput: { marginTop: 10, maxWidth: 150, minWidth: 130 },
+  tenantInitialAdminTooltip: {
+    alignSelf: 'flex-end'
+  },
+  editLimitButton: {
+    height: '42px',
+    alignSelf: 'center'
+  }
+}));
+const { setSnackbar } = actions;
+
 export const ExpandedTenant = (props: ExpandedTenantProps) => {
-  const { tenantId, onCloseClick } = props;
+  const { onCloseClick, tenant } = props;
+  const { name, id, device_limit, device_count, binary_delta } = tenant;
+
+  const [shouldDelete, setShouldDelete] = useState<boolean>(false);
+  const [newLimitForm, setNewLimitForm] = useState<boolean>(false);
+  const [newLimit, setNewLimit] = useState<number>(device_limit);
+  const [limitErrorText, setLimitErrorText] = useState<string>('');
+
+  const { device_count: spDeviceUtilization, device_limit: spDeviceLimit } = useSelector(getOrganization);
+  const { classes } = useStyles();
+  const dispatch = useDispatch<AppDispatch>();
+  const copyLinkToClipboard = () => {
+    const location = window.origin + '/ui';
+    copy(`${location}${generateTenantPathByName(name)}`);
+    dispatch(setSnackbar('Link copied to clipboard'));
+  };
+
+  const onChangeLimit = ({ target: { validity, value } }) => {
+    if (validity.valid) {
+      setNewLimit(value);
+      return setLimitErrorText('');
+    }
+    setLimitErrorText('Device limit');
+  };
+  const onNewLimitSubmit = async () => {
+    await dispatch(editTenantDeviceLimit({ id, name, newLimit: Number(newLimit) }));
+    setNewLimitForm(false);
+  };
+  const deleteTenant = () => {
+    dispatch(removeTenant({ id }));
+  };
+
   return (
     <Drawer onClose={onCloseClick} open={true} PaperProps={{ style: { minWidth: '67vw' } }} anchor="right">
-      Information for tenant with id {tenantId}
+      <div className="flexbox center-aligned space-between">
+        <div className="flexbox center-aligned">
+          <div className={`${classes.tenantTitle}`}>Tenant Information for {name}</div>
+          <IconButton onClick={copyLinkToClipboard} size="large">
+            <LinkIcon />
+          </IconButton>
+        </div>
+        <div className="flexbox center-aligned">
+          <Button onClick={() => setShouldDelete(true)}>Delete tenant</Button>
+          <IconButton onClick={onCloseClick} aria-label="close" size="large">
+            <CloseIcon />
+          </IconButton>
+        </div>
+      </div>
+      <Divider className="margin-bottom" />
+      <div className="margin-top">
+        <div className="flexbox">
+          <TwoColumns
+            setSnackbar={true}
+            style={{ width: '482px' }}
+            items={{
+              name,
+              ID: id,
+              'Initial admin user': 'customer@email.com'
+            }}
+          />
+          <div className={`${classes.tenantInitialAdminTooltip}`}>
```

Comment:
```suggestion
          <div className={classes.tenantInitialAdminTooltip}>
```
as long as there's no additional string content added